GitUI/GitUICommands.cs

```diff
@@ -534,6 +534,11 @@ bool Action()
                     }
 
                     using FormCommit form = new(this, commitMessage: commitMessage);
+
+                    // WORKAROUND: Marani Paolo 8/9/22
+                    // Ensure the form can be always displayed when owner is in a different screen
+                    form.FixForOwnerScreenLocation(owner);
```

Comment:
Only for Commit, should it be general when reading values?

Reply:
This should be private to the affected form. I.e. it should be done `OnLoad` or `OnActivate`.
Perhaps it should be generalized and be added to `GitExtensionsForm.RestorePosition()` or rather to `WindowPositionManager.FitWindowOnScreen()`.